indico/modules/events/registration/__init__.py

```diff
@@ -272,3 +272,8 @@ def _patch_google_wallet_ticket(registration, change, **kwargs):
         gwm = GoogleWalletManager(registration.event)
         if gwm.is_configured:
             gwm.patch_ticket_object(registration)
+
+
+def get_custom_ticket_qr_code_handlers():
+    """Get a dict containing custom ticket QR code handlers."""
+    return named_objects_from_signal(signals.event.registration.custom_ticket_qr_code_handler.send())
```

Comment:
This raises an exception in case multiple objects with the same name are registered.

We have code like this in other places to fail early at initialization time to immediately make it clear that something is wrong:

```py
@signals.core.app_created.connect
def _check_field_definitions(app, **kwargs):
    # This will raise RuntimeError if the field names are not unique
    get_contrib_field_types()
```

Hence, please add this here (this one should be in `__init__.py`) as well:

```py
@signals.core.app_created.connect
def _check_custom_qr_code_handlers(app, **kwargs):
    # This will raise RuntimeError if the handler names are not unique
    get_custom_ticket_qr_code_handlers()
```